gnomad/utils/transcript_annotation.py

```diff
@@ -323,28 +323,38 @@ def tx_filter_variants_by_csqs(
             lambda csq: hl.is_defined(csq.amino_acids) & (csq.amino_acids != "*")
         ]
 
-    keep_csqs = True
     if ignore_splicing:
-        if filter_to_csqs is not None:
-            filter_to_csqs = [csq for csq in filter_to_csqs if csq not in CSQ_SPLICE]
-        else:
-            filter_to_csqs = CSQ_SPLICE
-            keep_csqs = False
-
-    if filter_to_csqs is not None:
-        logger.info("Adding most severe consequence to VEP transcript consequences...")
-        ht = process_consequences(
-            ht, vep_root=vep_root, has_polyphen=include_polyphen_prioritization
+        logger.info("Filtering VEP consequences to exclude splice consequences...")
+        ht = ht.annotate(
+            **{
+                vep_root: ht[vep_root].annotate(
+                    transcript_consequences=ht[vep_root]
+                    .transcript_consequences.map(
+                        lambda csq: csq.annotate(
+                            consequence_terms=csq.consequence_terms.filter(
+                                lambda x: ~hl.literal(CSQ_SPLICE).contains(x)
+                            )
+                        )
+                    )
+                    .filter(lambda csq: hl.len(csq.consequence_terms) > 0)
```

Comment:
I actually started to add it to the vep utils, but then decided against it because that was clearly filtering based on most severe consequence, and I thought it might be confusing to have both.

Reply:
This fix is to make it do what was done in the original

Reply:
yeah, I keep going back and forth on whether this (filtering on any csq) is helpful to have, but maybe you're right that it might be too confusing